### PR TITLE
Add Vulnerability Management WG

### PR DESCRIPTION
**Pull request by emalm**
TODO before merge:
* Identify leads for WG
* Identify initial goals and scope for WG
* List main technical assets for WG

This PR supersedes #147: the content is identical, but it pulls from the `wg-vulnerability-management` branch on the CF-org community repo instead of the one on my personal fork so that the other TOC members can revise it.

### CODE REVIEW

File: toc/working-groups/vulnerability-management.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Vulnerability Management: Working Group Charter
+
+## Mission
+
+Provides discreet management of security vulnerabilities issues relevant for active CF projects.
+
+## Goals
+
+* Provides a single point of contact for security vulnerability reporting and management
+* Provides management of security vulnerability reports through to resolution, including but not limited to triage, reporter and team coordination, embargo negotiation, CVSS scoring, CVE assignments, pre-disclosure and disclosure
```
Comment (emalm) [inline toc/working-groups/vulnerability-management.md:10]:
```suggestion
* Provide a single point of contact for security vulnerability reporting and management.
* Provide management of security vulnerability reports through to resolution, including but not limited to triage, reporter and team coordination, embargo negotiation, CVSS scoring, CVE assignments, pre-disclosure and disclosure.
```

File: toc/working-groups/vulnerability-management.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## Scope
+
+* Triage incoming security vulnerability reports to [security@cloudfoundry.org](mailto:security@cloudfoundry.org)
+* Management of vulnerabilities through dedicated slack channels
+* When appropriate, negotiating suitable embargo periods with the reporter to afford component teams time to fix the issue before it becomes known publicly
+* When appropriate, assignment of CVE numbers to vulnerabilities/fixes
+* Publication of pre-disclosures to allow all CF editions time to adopt fixes for high/critical vulnerabilities before they become known publicly
+* Publication of disclosures
```
Comment (emalm) [inline toc/working-groups/vulnerability-management.md:19]:
```suggestion
* Triage incoming security vulnerability reports to [security@cloudfoundry.org](mailto:security@cloudfoundry.org).
* Manage vulnerabilities through dedicated slack channels.
* When appropriate, negotiate suitable embargo periods with the reporter to afford component teams time to fix the issue before it becomes known publicly.
* When appropriate, assign CVE numbers to vulnerabilities/fixes.
* Publish pre-disclosures to allow all CF distributions time to adopt fixes for high/critical vulnerabilities before they become known publicly.
* Publish disclosures of reported security vulnerabilities.
```

File: toc/working-groups/vulnerability-management.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+## Non-Goals
+
+* Not adding security related features to Cloud Foundry
```
Comment (emalm) [inline toc/working-groups/vulnerability-management.md:23]:
```suggestion
* Add security-related features to Cloud Foundry projects.
```

File: toc/working-groups/vulnerability-management.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+## Technical Lead(s): 
+- Paul Warren
+
+## Execution Lead(s):
+- Monty Ijzerman
+- Khushboo Soni
+- Matthias Essenpres
```
Comment (emalm) [inline toc/working-groups/vulnerability-management.md:31]: I think this is the correct spelling of this person's name, based on past emails, if I'm thinking of the right person.
```suggestion
- Mathias Essenpries 
```

File: toc/working-groups/vulnerability-management.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+* Not adding security related features to Cloud Foundry
+
+## Technical Lead(s): 
+- Paul Warren
+
+## Execution Lead(s):
+- Monty Ijzerman
+- Khushboo Soni
+- Matthias Essenpres 
+- Prateek Mehrotra
```
Comment (emalm) [inline toc/working-groups/vulnerability-management.md:32]: We should add GitHub handles for these members.

**Review by emalm — COMMENTED** [review]
I made some stylistic edits, but I think we also need some more concrete contact information for the leads.

File: toc/working-groups/vulnerability-management.md (hunk 4 above)
Comment (emalm) [inline toc/working-groups/vulnerability-management.md:31]:
```suggestion
- Mathias Essenpreis 
```

Comment (emalm) [reply toc/working-groups/vulnerability-management.md]: @stephanme or @loewenstein to confirm I have the spelling right this time 😅

**Comment by LeePorte** [comment]
We now have the majority of GitHub handles in place with only one person left to respond. I'm happy to not hold this up based on one response.

**Comment by emalm** [comment]
@LeePorte Sounds good to me to proceed with the current set of GitHub handles.

File: toc/working-groups/vulnerability-management.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+* Add security-related features to Cloud Foundry projects.
+
+## Technical Lead(s): 
+- [Paul Warren](https://github.com/paulcwarren)
+
+## Execution Lead(s):
+- [Monty Ijzerman](https://github.com/ijzerman)
+- [Khushboo Soni](https://github.com/skhushboo-vm)
+- Mathias Essenpreis
```
[the remark this answers is not in the document]
Comment (LeePorte) [reply toc/working-groups/vulnerability-management.md]: Thank you Sir

**Review by stephanme — APPROVED** [review]

**Review by loewenstein — APPROVED** [review]

File: toc/working-groups/vulnerability-management.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+* When appropriate, assign CVE numbers to vulnerabilities/fixes.
+* Publish pre-disclosures to allow all CF distributions time to adopt fixes for high/critical vulnerabilities before they become known publicly.
+* Publish disclosures of reported security vulnerabilities.
+
+## Non-Goals
+
+* Add security-related features to Cloud Foundry projects.
+
+## Technical Lead(s): 
+- [Paul Warren](https://github.com/paulcwarren)
```
Comment (dsboulder) [inline toc/working-groups/vulnerability-management.md:26]: I think @paulcwarren is trying to get out of this role he's been playing. Do we have other ideas in mind for who could do this in the OSS community?

**Review by dsboulder — CHANGES_REQUESTED** [review]

File: toc/working-groups/vulnerability-management.md (hunk 7 above)
Comment (LeePorte) [reply toc/working-groups/vulnerability-management.md:26]: I don't know if he's found anybody as yet.

Comment (paulcwarren) [reply toc/working-groups/vulnerability-management.md:26]: Sure.  Go ahead Lee.  For now, its me so merging is fine.

**Review by dsboulder — APPROVED** [review]

**Review by dsboulder — APPROVED** [review]

**Pull request merged**